docs/UsingCppInterOp.rst

```diff
@@ -67,33 +67,204 @@ that it was designed for.
 
 Further Enhancing the Dynamic/Automatic bindings in CPPYY
 =========================================================
+
+CppInterOp ties into the following major enhancements in cppyy:
```

Comment:
link to code/doc

Reply:
Linked to Paper "Efficient and Accurate Automatic Python Bindings with cppyy & Cling"